examples/mnist.py

```diff
@@ -100,25 +100,25 @@ def build_mlp(input_var=None):
     # Add a fully-connected layer of 800 units, using the linear rectifier, and
     # initializing weights with Glorot's scheme (which is the default anyway):
     l_hid1 = lasagne.layers.DenseLayer(
-            l_in_drop, num_units=800,
-            nonlinearity=lasagne.nonlinearities.rectify,
-            W=lasagne.init.GlorotUniform())
+        l_in_drop, num_units=800,
+        nonlinearity=lasagne.nonlinearities.rectify,
+        W=lasagne.init.GlorotUniform())
```

Comment:
I'm fine with everything but these hanging indentation changes, they were more readable before. After a long discussion of the wording of hanging indentation rules in PEP8, the pep8 project decided to not report E126 by default from version 1.6.0 on. We then included the pep8 version to use in `requirements-dev.txt`: https://github.com/Lasagne/Lasagne/pull/289

So if you upgrade your pep8, you should be able to undo these changes! Please amend the commit instead of adding a new one, thanks!